packages/php/blueprint/tests/Unit/ExportSchemaTest.php

```diff
@@ -35,7 +35,7 @@ public function get_mock( $partial = false ) {
 	 * with the built-in exporters.
 	 */
 	public function test_it_uses_exporters_passed_to_the_constructor() {
-		$empty_exporter     = new EmptySetSiteOptionsExporter();
+		$empty_exporter = new EmptySetSiteOptionsExporter();
```

Comment:
To fix the lint error:

```suggestion
		$empty_exporter     = new EmptySetSiteOptionsExporter();
```